frontend/its-our-call/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@types/react-redux": "^5.0.15",
     "d3": "^5.1.0",
+    "hoek": "^5.0.3",
```

Comment:
What is hoek used for? I don't see it imported/required anywhere?

Reply:
It's a nested-dependency: 
`hoek->boom->cryptiles->hawk->request->node-pre-gyp->fsevents->react-scripts-ts`
I've never used it directly but it's a widely used utility library that recently reported a security vulnerability, and they're suggesting everyone update to version 4.2 or higher. 
https://nodesecurity.io/advisories/566

Unfortunately, some of the packages we use have the version pinned below version 5, so we can't do much other than wait for them to update their package.
Fortunately, it's only a vulnerability if we're running our dev server, which will not happen for us in production (webpack ftw), so we should be fairly safe.

But regardless I'm adding it directly as a dependency in case that there are any sub-dependency updates with broad ranges, in which case this explicit dependency should break the tie.

Reply:
Ah OK, thanks for the info.